### ISSUE TITLE
Content without excerpt/summary

### ISSUE THREAD

**Issue by NuLL3rr0r** [gohugoio/hugo#4698]
I was looking at [this blog post](https://www.codeooze.com/webdesign/hugo-toc/) and I would like to allow only some of my blogs posts to have TOC; e.g. by setting in the front matter:

```
+++
...
toc = true
...
+++
```

Now I would like the post to look like this when the user clicks on Read More... button:

```
{{ .Summary }}

{{ .TableOfContents }}

{{ Rest of the .Content without summary part }}
```

Looking at [the docs](https://gohugo.io/content-management/toc/) there seems to be only {{ .Summary }}, {{ .TableOfContents }}, and {{ .Content }}. It would be nice if Hugo allows more control by introducing a new variable which does not contain the {{ .Summary }} part.

**Comment by NuLL3rr0r** [comment]
Well, it seems after I opened the issue, I figured it's possible to replace the summary like this:

```
        {{ if .Params.toc }}
        {{ .Summary }}
        {{ .TableOfContents }}
        {{ replace .Content .Summary "" | safeHTML }}
        {{ else }}
        {{ .Content }}
        {{ end }}
```

Though it's a bit ugly to do {{ replace .Content .Summary "" | safeHTML }} it solves it for me. So, I'm not sure if I should close this issue or leave it as a feature request. Feel free to close it if you think it's redundant to have another variable to achieve a similar results.

Thanks!
